app/misc.py

```diff
@@ -1411,9 +1412,11 @@ def metadata_to_dict(metadata):
 LOG_TYPE_DISABLE_INVITE = 48
 LOG_TYPE_DISABLE_REGISTRATION = 49
 LOG_TYPE_ENABLE_REGISTRATION = 50
-
 LOG_TYPE_USER_UNBAN = 51
 
+LOG_TYPE_CLOSE_REPORT = 52
+LOG_TYPE_REOPEN_REPORT = 53
+LOG_TYPE_CLOSE_RELATED_REPORT = 54
```

Comment:
These should be prefixed LOG_TYPE_REPORT_xxx to keep it consistent with the sub action log types.